bitcoin/src/blockdata/witness.rs

```diff
@@ -225,8 +225,7 @@ impl Encodable for Witness {
         let content_with_indices_len = self.content.len();
```

Comment:
The 9fe420d2210bfcfab4d0913c9dd94f633496f95a commit message makes it sound like getting the length is impossible without returning it. That's a complete BS.

Instead the message should be something like "Returning `usize` is more convenient for consumers that need to add up the length of multiple encoded items."